marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/writer.spec.tsx

```diff
@@ -21,7 +21,7 @@ describe('Entity Modeling: Writer Role', () => {
     cy.contains(Application.title);
     console.log(Cypress.env('mlHost'));
     cy.loginAsTestUserWithRoles("hub-central-entity-model-reader", "hub-central-entity-model-writer", "hub-central-saved-query-user").withRequest()
-    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
+    cy.waitUntil(() => toolbar.getModelToolbarIcon().should('have.css', 'cursor', 'pointer')).click();
```

Comment:
Doing this to address this e2e test failure on windows. 
<img width="1322" alt="model-writer-fail" src="https://user-images.githubusercontent.com/32427979/90195755-23d57c00-dd7f-11ea-87a0-766a9dc888ba.png">

I believe this is due to toolbar component using inline style instead of CSS. 
https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad

Reply:
I doubt if that is the case. Other tool icons are clicked the same way and in windows, in this instance, load test didnt fail there. To me it seems to be the async issue. We are using wait(500) in withRequest() and the websocket get request might not have completed in that time resulting in that click timeout error. Will try to replace explicit wait with something else to ensure overview page is completely loaded with all XHR requests complete.

Reply:
It’s an intermittent failure where it sometimes fails and sometimes passes. The log output above shows the style has not been set correctly. But if you see the attached image below of where the video fails, the icon doesn’t have the style that’s in the error log.
<img width="1322" alt="video-fail" src="https://user-images.githubusercontent.com/32427979/90206213-f8ab5680-dd97-11ea-9373-d980ee1057aa.png">